### ISSUE TITLE
Are private unified margin WS topics working?

### ISSUE THREAD

**Issue by tiagosiebler** [tiagosiebler/bybit-api#175]
Private topics for unified margin might not be working yet due to the unknown `3303001` error code. Needs investigating

**Comment by twxia** [comment]
Hi @tiagosiebler it seems that if I update to `50000`, it works somehow but it is not that stable. Sometimes I got `error sign!` message (retCode: `10004`) 
https://github.com/tiagosiebler/bybit-api/blob/9b845eede3f656cc3b579e6f1b98f7e0ede8ca53/src/websocket-client.ts#L500

**Comment by wiiiwlabs** [comment]
```
 const wsClient = new WebsocketClient(
    {
      key: key,
      secret: secret,
     market: 'unifiedPerp',
    },
    logger
  );
```
```
wsClient.subscribe('user.position.unifiedAccount', true);
```

**Comment by tiagosiebler** [comment]
Fixed in #183, released in v3.1.1. Thanks @twxia !
